extension/public/popup.html

```diff
@@ -74,23 +74,6 @@ <h3>Keywords</h3>
                 </div>
             </div>
         </div>
-        <div class="bottom">
-            <div class="backend">
-                <span>Connect to our backend</span>
-                <label class="switch tooltip">
-                    <input type="checkbox" checked>
-                    <span class="slider round"></span>
-                    <span class="tooltiptext">
```

Comment:
`tooltiptext` is no longer used and can be removed from the CSS (unless it's used anywhere else?).

Reply:
Sorry this PR is not yet approved (I clicked the wrong button!)

Reply:
Yeah, you are correct. I first thought that the slider had been used by other sections and remained it in the CSS file. However, I then forgot to delete tooltips, lol